src/schematools/permissions/db.py

```diff
@@ -393,16 +416,12 @@ def _execute_grant(session, grant_statement, echo=True, dry_run=False):
     We don't want to break out of the session just because a table, column, or user doesn't
     exist.
     """
-
     status_msg = "Skipped" if dry_run else "Executed"
     sql_statement = f"""
```

Comment:
The airflow grants operator is calling in to schematools from here: https://github.com/Amsterdam/dataservices-airflow/blob/7d9db2e427e79c622a3bd5479f415bf2dd69e1d5/src/plugins/postgres_permissions_operator.py#L163

Could this break, after removing the postgres exception handler?

Reply:
If needed, I can do some testing for you, I have a working development Airflow.

Reply:
that depends on the versions of the schemas that it is called with. Ive run it against the latest versions without error